src/test/java/TaskListTest.java

```diff
@@ -0,0 +1,88 @@
+import Mario.TaskList;
+import Mario.Tasks.Task;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Perhaps an alternative to this statement could be to import each and every Assertion you have used individually?